src/map/utils.es

```diff
@@ -1,49 +1,29 @@
-import { getMedia, media } from '../utils';
+import { LngLatBounds } from 'mapbox-gl';
 
-const controlsProps = [
-  'doubleClickZoom',
-  'boxZoom',
-  'touchZoom',
-  'dragging',
-  'tap',
-  'zoomControl',
-];
 
-const viewProps = [
-  'locked',
-  'lockedMobile',
-  'noAttribution',
-];
+export const getBoundsCenter = (bounds) => ([
+  (bounds[0][0] + bounds[1][0]) / 2,
+  (bounds[0][1] + bounds[1][1]) / 2,
+]);
 
-export const getMapOptions = (node, { noAttribution, locked, lockedMobile }) => {
-  const isMobile = !getMedia(node, media.mediaM);
-  const isControlsEnabled = isMobile ? !lockedMobile : !locked;
-
-  const base = controlsProps.reduce((acc, prop) => {
-    acc[prop] = isControlsEnabled;
-    return acc;
-  }, {});
-
-  return { ...base, scrollWheelZoom: false, attributionControl: !noAttribution };
-};
-
-export const getTileUrl = (credentials) => (
-  credentials
-    ? `https://api.maptiler.com/maps/streets/256/{z}/{x}/{y}.png?key=${credentials.key}`
-    : 'http://{s}.tile.osm.org/{z}/{x}/{y}.png'
+export const getStyle = (credentials) => (
+  `https://api.maptiler.com/maps/075de9e7-0503-4e15-87af-7b99e074604c/style.json?key=${credentials.key}`
 );
 
-export const getTileOptions = (credentials) => {
-  const osm = '<a href="http://osm.org/copyright" target="_blank" rel="noopener noreferrer">&copy; OpenStreetMap contributors</a>';
-  if (!credentials) return { attribution: osm };
+export const getLngLatBounds = (bounds) => {
+  if (!Array.isArray(bounds) || !bounds.length) return undefined;
 
-  const attribution = `
-    <a href="https://www.maptiler.com/copyright/" target="_blank" rel="noopener noreferrer">&copy; MapTiler</a> ${osm}
-  `;
+  const initial = new LngLatBounds(bounds[0], bounds[0]);
```

Comment:
same coord twice?

Reply:
yep. got this part from mapbox docs https://docs.mapbox.com/mapbox-gl-js/example/zoomto-linestring/

Reply:
leave a comment